### PR TITLE
Replace pyswsssdk with swsscommon

### PR DESCRIPTION
**Pull request by liuh-80**
#### What I did
Replace sonic-py-swsssdk with sonic-swss-common.

#### How I did it
Update all sonic-py-swsssdk to use sonic-swss-common or sonic-py-common.

#### How to verify it
Pass all UT and E2E test.

### CODE REVIEW

**Comment by liuh-80** [comment]
This PR depends on another PR to merge first:
https://github.com/Azure/sonic-buildimage/pull/11215

File: config/main.py [hunk 1]
```diff
@@ -6539,7 +6539,7 @@ def rate():
 @click.argument('rates_type', type=click.Choice(['all', 'port', 'rif', 'flowcnt-trap']), default='all')
 def smoothing_interval(interval, rates_type):
     """Set rates smoothing interval """
-    counters_db = swsssdk.SonicV2Connector()
+    counters_db = SonicV2Connector()
```
Comment (liuh-80) [inline config/main.py:6542]: Following API missing in sonic-swss-common:

    int64_t SonicV2Connector_Native::set(const std::string& db_name, const std::string& _hash, const std::string& key, const int val, bool blocking=false);

So this PR need wait for swss-common add the missing API first.

File: README.md [hunk 2]
```diff
@@ -26,7 +26,6 @@ Currently, this list of dependencies is as follows:
 - libyang-cpp_1.0.73_amd64.deb
 - python3-yang_1.0.73_amd64.deb
 - redis_dump_load-1.1-py3-none-any.whl
-- swsssdk-2.0.1-py3-none-any.whl
```
Comment (qiluo-msft) [inline README.md:29]: >swsssdk [](http://example.com/codeflow?start=2&length=7)

Add "python-swsscommon_1.0.0_amd64.deb" into the list?

**Review by qiluo-msft — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (liuh-80) [reply README.md:29]: Fixed.

**Comment by liuh-80** [comment]
The depends PR merged, so this PR is ready for review.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**